pass/add_creds.sh

```diff
@@ -167,6 +167,16 @@ github_ssh() {
   ssh_keys "${project_name}" "${site}" "${user}"
 }
 
+matrix() {
```

Comment:
"matrix" command should be added to the help() function.

Reply:
done!

Reply:
Are you sure you have pushed the change? I only see: https://github.com/eclipse-cbi/ci-admin/compare/b685abc222264756d0b1f4215ada298da4017c61..87ec1a878d8e4ebb785fc4dda324bd0defbfbd37

Reply:
Looks better now! :+1: